### PR TITLE
Added conv ai 2 (Again)

### PR DESCRIPTION
**Pull request by rkc007**
The original PR -> https://github.com/huggingface/datasets/pull/1383

Reason for creating again - 

The reason I had to create the PR again was due to the master rebasing issue. After rebasing the changes, all the previous commits got added to the branch.

### CODE REVIEW

**Comment by tanmoyio** [comment]
Looking perfect to me, need to rerun the tests

**Comment by rkc007** [comment]
Thanks, @tanmoyio. 
How do I rerun the tests? Should I change something or push a new commit?

**Comment by tanmoyio** [comment]
@rkc007 you don't need to rerun it, @lhoestq @yjernite will rerun it, as there are huge number of PRs in the queue it might take lil bit of time.

**Comment by abhishekkrthakur** [comment]
ive just re-run the tests

**Comment by rkc007** [comment]
Thank you @abhishekkrthakur. Can you please rerun it again? It seems something was broken in CI during the previous test.

File: datasets/conv_ai_2/conv_ai_2.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+_DESCRIPTION = """\
+ConvAI is a dataset of human-to-bot conversations labelled for quality. \
+This data can be used to train a metric for evaluating dialogue systems. \
+Moreover, it can be used in the development of chatbots themselves: it contains the information \
+on the quality of utterances and entire dialogues, that can guide a dialogue system in search of better answers.
+"""
+
+_URL = "https://github.com/DeepPavlov/convai/raw/master/2018/data/summer_wild_evaluation_dialogs.json"
+
```
Comment (lhoestq) [inline datasets/conv_ai_2/conv_ai_2.py:16]: Could you also add the _CITATION here ?

File: datasets/conv_ai_2/README.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+### Data Fields
+
+- dialog_id : specifies the unique ID for the dialogs.
+- dialog : Array of dialogs.
+- bot_profile : Bot annotated response that will be used for evaluation.
+- user_profile : user annoted response that will be used for evaluation.
+- eval_score : (1, 2, 3, 4, 5;) how does an user like a conversation.
+- profile_match : 0, 1.0; an user is given by two profile descriptions (4 sentences each), one of them is the one given to the bot it had been talking to, the other one is random; the user needs to choose one of them.
+
```
Comment (lhoestq) [inline datasets/conv_ai_2/README.md:121]: can you mention that some missing values are replaced with -1 ?

File: datasets/conv_ai_2/conv_ai_2.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        datasets.BuilderConfig(
+            name="conv_ai_2",
+            version=datasets.Version("1.0.0"),
+            description="Full training set",
+        ),
+    ]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
```
Comment (lhoestq) [inline datasets/conv_ai_2/conv_ai_2.py:32]: then you can do citation=_CITATION here

**Review by lhoestq — COMMENTED** [review]
This looks very clean thanks !

I left my final comments :)

**Comment by rkc007** [comment]
@lhoestq Sorry for the mess. I don't know why this keeps on happening. I tried step by step process of updating the PR but seems something is wrong. This happened for 2nd time with the same PR. Apologies for that. 

New PR -> https://github.com/huggingface/datasets/pull/1527
Also, I fixed everything in the new PR.

**Pull request closed without merging**